### ISSUE TITLE
PreTrained FlowNetS with BatchNormalization

### ISSUE THREAD

**Issue by sparviero19** [ClementPinard/FlowNetPytorch#12]
Hi, 

is there a pretrained pytorch model of the batch normalized version of FlownetS? 
Thanks

**Comment by ClementPinard** [comment]
Pretrained models will come when I manage to get ~3 EPE. Still not reaching it, my best so far is 3.9
Stay tuned !

**Comment by taineleau** [comment]
How long does it take to achieve 3.9? Could you also share me with the server config?

**Comment by ClementPinard** [comment]
Hello, sorry for the late answer, convergence problems are now solved, documentation and networks can now be found [here](https://github.com/ClementPinard/FlowNetPytorch#training-results)

Clément
